components/writers/src/main/java/org/datacleaner/extension/output/CreateExcelSpreadsheetAnalyzer.java

```diff
@@ -71,11 +74,14 @@ public void validate() {
         if (sheetName.indexOf(".") != -1) {
             throw new IllegalStateException("Sheet name cannot contain dots (.)");
         }
+
+        if (file.exists() && !overwriteFile) {
```

Comment:
This also means that this class is still missing a validate method which will fail in the scenario where overwriteFile is false and the file exists.